ksql-engine/src/main/java/io/confluent/ksql/internal/KsqlEngineMetrics.java

```diff
@@ -66,6 +75,15 @@ public KsqlEngineMetrics(final String metricGroupPrefix, final KsqlEngine ksqlEn
     this.numIdleQueries = configureIdleQueriesSensor(metrics);
     this.messageConsumptionByQuery = configureMessageConsumptionByQuerySensor(metrics);
     this.errorRate = configureErrorRate(metrics);
+    this.createdQueriesCount = configureNumActiveQueriesForGivenState(metrics, State.CREATED);
```

Comment:
Rather that having explicit lines for each state, why not just use:

```
Arrays.stream(State.values())
        .forEach(state -> configureNumActiveQueriesForGivenState(metrics, state));
```

Which means any new or changed states will automatically be added...

Reply:
Made the change.